tests/packagedcode/test_phpcomposer.py

```diff
@@ -58,35 +58,55 @@ def test_parse_person(self):
     def test_parse_atimer(self):
         test_file = self.get_test_loc('phpcomposer/a-timer/composer.json')
         expected_loc = self.get_test_loc('phpcomposer/a-timer/composer.json.expected')
-        package = phpcomposer.parse(test_file)
+        packages = list(phpcomposer.parse(test_file))
+        assert len(packages) > 0
+        package = packages[0]
         self.check_package(package, expected_loc, regen=False)
 
     def test_parse_framework(self):
         test_file = self.get_test_loc('phpcomposer/framework/composer.json')
         expected_loc = self.get_test_loc('phpcomposer/framework/composer.json.expected')
-        package = phpcomposer.parse(test_file)
+        packages = list(phpcomposer.parse(test_file))
+        assert len(packages) > 0
+        package = packages[0]
         self.check_package(package, expected_loc, regen=False)
 
     def test_parse_slim(self):
         test_file = self.get_test_loc('phpcomposer/slim/composer.json')
         expected_loc = self.get_test_loc('phpcomposer/slim/composer.json.expected')
-        package = phpcomposer.parse(test_file)
+        packages = list(phpcomposer.parse(test_file))
```

Comment:
Why testing a single package? and not all of them?

Reply:
I will update the PackageTester class to be able to check lists of packages instead of single packages